plugins/org.eclipse.fordiac.ide.systemmanagement.ui/plugin.xml

```diff
@@ -588,7 +588,28 @@
             </instanceof>
          </filterExpression>
       </commonFilter>
-  </extension>  
+  </extension>
+      <extension
+            point="org.eclipse.ui.navigator.navigatorContent">
+         <navigatorContent
+               activeByDefault="true"
+               id="org.eclipse.fordiac.ide.systemmanagement.ui.systemexplorer.sortOnly"
+               name="name"
+               priority="normal"
+               sortOnly="true">
+            <commonSorter
```

Comment:
why is this sorter not part of the navigator content above?

Reply:
This is a sortOnly NavigationContext that overrides sorting regardless of underlying NavigationContext
This is aimed at `org.eclipse.ui.navigator.resourceContent` in this case

Reply:
thx for the clarification.